Code/Framework/AzFramework/Platform/Windows/AzFramework/Windowing/NativeWindow_Windows.cpp

```diff
@@ -16,6 +16,8 @@
 #include <AzCore/std/containers/array.h>
 #include <AzCore/std/string/conversions.h>
 
+#include <dbt.h>
+
```

Comment:
Sure is a pity to be bringing in DBT.h for  one const value, 
since it brings in guiddef.h and string.h from MSFT.

I checked and it does NOT bring in windows platform headers, (or windows.h or anything like that) so its *probably* safe. 

So nothing to do here, except define our version which has the same value.  But probably not worth it for the confusion (nothing to do here).